contrib/docker_compose_workers/README.md

```diff
@@ -73,6 +73,21 @@ redis:
 
 This assumes that your Redis service is called `redis` in your Docker Compose file.
 
+When using [Redis Sentinel](https://redis.io/docs/management/sentinel/) the following configuration can be used instead:
+
+```yaml
+redis:
+  enabled: true
+  sentinel: true
+  service_name: service_name
+  servers:
```

Comment:
It might be more "correct" to call these `sentinels` (as you have it in `config_documentation.md`)